products/db-checkbook/build_scripts/build.py

```diff
@@ -88,7 +85,7 @@ def _group_checkbook(data: pd.DataFrame = None) -> pd.DataFrame:
     """
     :return: checkbook nyc data grouped by capital project
     """
-    if not data:
+    if data.empty:
```

Comment:
Why this check? In what case will empty data be fed in?

Reply:
There was a version of testing where I added default params for all the build fns in order to pass in test data but ended up scrapping it. Missed this one, will remove

Reply:
still wanted to drop this?